src/actions/index.js

```diff
@@ -9,10 +10,23 @@ export const DECREMENT = 'DECREMENT';
 // logic of updating the central Redux store. That
 // is left to the reducer(s).
 
-export const increment = () => {
-  // Fill in this function
+export const increment = num => {
+  return {
+    type: INCREMENT,
+    payload: num
+  }
 };
 
-export const decrement = () => {
-  // Fill in this function
+export const decrement = num => {
+  return {
+    type: DECREMENT,
+    payload: num
+  }
 };
+
+export const incrementIfOdd = num => {
+  return {
+    type: INCREMENTODD,
+    payload: num
+  }
```

Comment:
this is good practice for making your own action creators, but I'll touch on why this one isn't necessary in Counter.js